wgpu-core/src/device/life.rs

```diff
@@ -355,7 +355,10 @@ impl<B: GfxBackend> LifetimeTracker<B> {
                     #[cfg(feature = "trace")]
                     trace.map(|t| t.lock().add(trace::Action::DestroyRenderBundle(id.0)));
                     let res = hub.render_bundles.unregister_locked(id.0, &mut *guard);
-                    self.suspected_resources.add_trackers(&res.used);
+
+                    if let Some(res) = res {
```

Comment:
nit: can we just combine this as `if let Some(res) = hub.render_bundles.unregister_locked(..) { ... }` here and in other cases?